source/Calamari.CloudAccounts/AwsEnvironmentGeneration.cs

```diff
@@ -171,6 +182,40 @@ async Task<bool> PopulateSuppliedKeys()
                 return true;
             }
 
+            if (!string.IsNullOrEmpty(roleArn))
```

Comment:
AWS Auth always breaks my brain. Is it safer to just check the oidcJwt here? It's a little hard to distinguish this variable RoleArn from the AssumedRoleArn on the surface. No need to change it if you disagree/have a green build.